charts/vela-core/templates/cluster-gateway/job-patch.yaml

```diff
@@ -89,6 +89,7 @@ spec:
           - --secret-name={{ template "kubevela.fullname" . }}-cluster-gateway-tls-v2
           - --cert-name=tls.crt
           - --key-name=tls.key
+          - --cert-gen-kubeconfig={{ .Values.admissionWebhooks.patch.certGenKubeConfig }}  
```

Comment:
Why do we need this?

Reply:
Checked the description, this PR is about to fix #6098. But for now user still can't use this for we can't mount the kubeconfig to this Job. We should add a volume related argument to the chart. @yashvardhanmishra